keps/sig-cluster-lifecycle/wgs/2141-instance-specific-componentconfig/kep.yaml

```diff
@@ -0,0 +1,25 @@
+title: Instance-Specific ComponentConfig
+authors:
+  - mtaufen
+  - knabben
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cluster-lifecycle
+  - sig-api-machinery
+  - sig-architecture
+  - wg-component-standard
+reviewers:
+  - sttts
+  - stealthybox
+  - liggitt
+  - rosti
+  - neolit123
+  - obitech
+approvers:
+  - sttts
+  - stealthybox
+  - liggitt
+editor: TBD
+creation-date: 2020-01-08
+last-updated: 2020-09-01
+status: implementable
```

Comment:
Since this is being merged as implementable do you want to say what release you are targetting or it is tbd?

Reply:
It should be determined by individual components that implement this KEP.
I think we can pick one of kubelet or kube-proxy for 1.21. @knabben WDYT?

Reply:
kubeadm is sorely in need of a solution for kubelet FWIW. Currently pet instance "config'" is only flags, which is problematic with kubelet instance specific flags (like provider id) being deprecated.

I'm not sure if these efforts will ultimately be related though, I think they had specific concerns around upgrade ... cc @neolit123